### ISSUE TITLE
finetune my own dataset problem

### ISSUE THREAD

**Issue by zhangdengp** [mzolfaghari/ECO-efficient-video-understanding#26]
when i finetune my own dataset ,loss=0
why?
I1115 13:17:15.940382 25582 solver.cpp:241] Iteration 20, loss = 0
I1115 13:17:15.940572 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:17:15.940580 25582 solver.cpp:665] Iteration 20, lr = 0.001
I1115 13:18:19.726904 25582 solver.cpp:241] Iteration 40, loss = 0
I1115 13:18:19.727092 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:18:19.727100 25582 solver.cpp:665] Iteration 40, lr = 0.001
I1115 13:19:23.873339 25582 solver.cpp:241] Iteration 60, loss = 0
I1115 13:19:23.873425 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:19:23.873433 25582 solver.cpp:665] Iteration 60, lr = 0.001
I1115 13:20:29.114048 25582 solver.cpp:241] Iteration 80, loss = 0
I1115 13:20:29.114857 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:20:29.114867 25582 solver.cpp:665] Iteration 80, lr = 0.001
I1115 13:21:34.218885 25582 solver.cpp:241] Iteration 100, loss = 0
I1115 13:21:34.218997 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:21:34.219003 25582 solver.cpp:665] Iteration 100, lr = 0.001
I1115 13:21:35.014933 25582 solver.cpp:273] Maximum resident set size is: 1352040 kb
I1115 13:21:35.015017 25582 solver.cpp:282] cuda meminfo: used 3957 MB, of 4037 MB
I1115 13:22:39.643602 25582 solver.cpp:241] Iteration 120, loss = 0
I1115 13:22:39.643775 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:22:39.643785 25582 solver.cpp:665] Iteration 120, lr = 0.001
I1115 13:23:45.116634 25582 solver.cpp:241] Iteration 140, loss = 0
I1115 13:23:45.116822 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:23:45.116832 25582 solver.cpp:665] Iteration 140, lr = 0.001
I1115 13:24:50.277351 25582 solver.cpp:241] Iteration 160, loss = 0
I1115 13:24:50.277442 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:24:50.277449 25582 solver.cpp:665] Iteration 160, lr = 0.001
I1115 13:25:55.094033 25582 solver.cpp:241] Iteration 180, loss = 0
I1115 13:25:55.094105 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:25:55.094113 25582 solver.cpp:665] Iteration 180, lr = 0.001
I1115 13:26:59.386559 25582 solver.cpp:241] Iteration 200, loss = 0
I1115 13:26:59.386631 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:26:59.386636 25582 solver.cpp:665] Iteration 200, lr = 0.001
I1115 13:28:04.782306 25582 solver.cpp:241] Iteration 220, loss = 0
I1115 13:28:04.782454 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:28:04.782464 25582 solver.cpp:665] Iteration 220, lr = 0.001
I1115 13:29:09.191298 25582 solver.cpp:241] Iteration 240, loss = 0
I1115 13:29:09.191468 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
I1115 13:29:09.191490 25582 solver.cpp:665] Iteration 240, lr = 0.001
I1115 13:30:13.419653 25582 solver.cpp:241] Iteration 260, loss = 0
I1115 13:30:13.419807 25582 solver.cpp:256]     Train net output #0: loss = 0 (* 1 = 0 loss)
